docs/sphinx-docs/source/user/RELEASE.rst

```diff
@@ -13,6 +13,71 @@ Release Notes
 Features
 ========
 
+New in Version 5.0.6
+--------------------
+This is a small bug release which fixes some issues reported in earlier versions
```

Comment:
Historically, we have bulleted the highlighted changes so this should be something like:

```
 * The failure to start when installing on a new system due to issues finding the config file has been fixed
 * The startup speed has been improved.
 * The paracryatline models, which have been labelled as "under review" since 2018, have been corrected (bcc and fcc) and the documentation completely reworked (bcc, fcc, and sc).
 * Plots now properly support custom names in the legend, 
 * The LM optimizer failure on GPUs or when the starting value is outside the min/max range has been fixed
 * A problem with intermittent blanking of plots has been fixed
 * A number of default values have been changed to be more reasonable
 * A number of issues in the documentation have been corrected and/or updated.
```

Reply:
Yes, I am aware of that as well as the fact that last time we decided to break things out into documentation, features, bugs, and infrastructure. I considered this for a while as I agree it is very important to be consistent. However I chose to deviate slightly (while still maintaining the general style) as this is the very first time we are trying to follow strict semantic versioning and thus limiting ourselves to a pure bug fix -- so in principle we have argued that EVERYTHING is a bug ... though there is a lot of grey areas and one could even quibble that maybe some are beyond grey?  Still I decided that sticking with the spirit of the release all would be bugs so no need for a specific section.

The bulleted highlights last time was partly due to the shear number of them. This release is much more restricted and focused and since all the updates are fairly cleanly stated in a small bullet set already I thought it might be better to use a narrative approach. Otherwise we are just making two copies of a third or so of the full bulleted list?  I also considered **not** including **any** highlights on the grounds again that the list is relatively short already. In the end I chose to include some highlights because I felt there were definitely some significant difference in relevance to many of ours users and felt it might be useful to highlight that for them. Looking at past releases, this is not totally inconsistent as we have mixed narrative and bullets in various sections based on what seemed appropriate for the content of that particular release.
I am happy to do what the consensus is on this but since these were deliberate choices I thought I should lay out my reasons for discussion before making changes. I would propose that we merge once I fix the typos and release to get the beta out sooner than later for testing and sort out what we think the correct answer to these should be.